core/trino-main/src/main/java/io/trino/execution/scheduler/EventDrivenFaultTolerantQueryScheduler.java

```diff
@@ -1760,6 +1765,7 @@ private StagePartition getStagePartition(int partitionId)
         private final Set<TaskId> runningTasks = new HashSet<>();
         private final Set<PlanNodeId> finalSelectors = new HashSet<>();
         private final Set<PlanNodeId> noMoreSplits = new HashSet<>();
+        private boolean taskScheduled;
```

Comment:
This is not a great name; at this point task is not scheduled, just merly put in the scheduling queue for scheduling late. Let's keep for now as  it at least is in-line with `PrioritizedScheduledTask`

Reply:
Yeah I was using `taskCreated`, but @arhimondr suggested to change to `taskScheduled`:)